src/Root.re

```diff
@@ -1,14 +1,43 @@
+open Dom.Storage2;
+open ThemeContext;
+
+let themeStorageKey = "theme";
+
+let themeToString =
```

Comment:
I think it would be better to collocate those functions with the type `theme` from `ThemeContext`, what do you think?

Reply:
Sure. I actually started with them there, but moved them here later for some reason (probably in order to not change more than one file in the PR).
Moved them back to ThemeContext now.